code/modules/vehicles/cars/clowncar.dm

```diff
@@ -76,11 +76,13 @@
 		L.visible_message("<span class='warning'>[src] rams into [L] and sucks [L.p_them()] up!</span>") //fuck off shezza this isn't ERP.
 		mob_forced_enter(L)
 		playsound(src, pick('sound/vehicles/clowncar_ram1.ogg', 'sound/vehicles/clowncar_ram2.ogg', 'sound/vehicles/clowncar_ram3.ogg'), 75)
+		log_combat(src, M, "sucked up")
```

Comment:
sucked by the clown :^)

Reply:
succ by the clown

Reply:
vored by the jester

Reply:
Shitposting aside, this will log the clown car as the attacker.

Reply:
there's no other way because there can be multiple drivers. Unless you log all the drivers or keep track of who did the last movement

Reply:
Yeah, that would be pretty messy. I think it's fine this way since it's extremely unlikely that a non antag clown would somehow grab the car and then run over people at arrivals. Maybe I should add logging to who enters the car without getting sucked into it?